packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -674,9 +713,21 @@ export default class PciInstancesAddController {
     );
   }
 
+  onInstanceSubmit() {
+    this.isLocalPrivateModeLocalZone = undefined;
+    this.isCreatingNewPrivateNetwork = undefined;
```

Comment:
**nitpick:** Prefer `null` for resetting the value. `null` is for manually emptying the value and `undefined` signifies the developer has forgot to declare/initialise the variable.
```suggestion
    this.isLocalPrivateModeLocalZone = null;
    this.isCreatingNewPrivateNetwork = null;
```